ci/build.yml

```diff
@@ -133,6 +133,37 @@ core fw btconly production build:
       - firmware-T2T1-btconly-production-*.*.*-$CI_COMMIT_SHORT_SHA.bin
     expire_in: 1 week
 
+core fw R debug build:
+  stage: build
+  <<: *gitlab_caching
+  needs: []
+  variables:
+    TREZOR_MODEL: "R"
```

Comment:
Shouldn't we use model T2B1?

Reply:
We theoretically could, but then we would also need to use `T2T1` in all cases where using `TREZOR_MODEL: T`. This is just an internal variable without public visibility, so no need to do that (now)

Reply:
Eventually we should but let's do it outside of this PR which is already very large.